config/locales/client.en.yml

```diff
@@ -2132,7 +2132,7 @@ en:
           confirm: You have a new topic draft saved, which will be overwritten if you create a linked topic.
         reply_as_new_group_message:
           label: Reply as new group message
-          desc: Create a new private message with the same recipients
+          desc: Create new message starting with same recipients
```

Comment:
Not to bikeshed here, but I find "starting with" more awkward than "with the same"

How strongly do you feel about this? I'm happy to remove "private"

Reply:
Maybe it should "starting with the same recipients"? The idea is that it evokes the fact that you can still add/remove recipients to your new message after selecting this option. We don't have much space.

If you find this option or it doesn't evoke this to you, then sure we can change it back but remove "private". Or maybe you have a better idea?

Reply:
I prefer "Create new message with same recipients" but I'll approve and leave it up to you :)